### PR TITLE
EthnicityForm

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**
EthnicityForm view is captured in this component is added, the functionality of Ethnicityform is shown in storybook.

### CODE REVIEW

**Comment by SashaKyz** [comment]
Can one of the admins verify this patch?

File: src/test/Ethnicity/EthnicityForm_test.js [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+        const detailSelect = component.find('SelectField');
+        expect(detailSelect.exists()).toEqual(true);
+        expect(detailSelect.props().label).toEqual('');
+        expect(detailSelect.props().value).toEqual('');
+        expect(detailSelect.props().id).toEqual(
+          'participant-123-ethnicity-detail'
+        );
+      });
+
+      it('sets the value of the selct to the current ethnicityDetail', () => {
```
Comment (edgibbs) [inline src/test/Ethnicity/EthnicityForm_test.js]: small typo 'selct' versus 'select'

File: src/test/Ethnicity/EthnicityForm_test.js [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+    });
+
+    describe('ethnicityDetailOptions select field', () => {
+      it('does not render if latioOrigin is not set to Yes', () => {
+        const component = renderPersonEthnicityForm({ latinoOrigin: '' });
+        const detailSelect = component.find('SelectField');
+        expect(detailSelect.exists()).toEqual(false);
+      });
+
+      it('renders with no label if latioOrigin is set to Yes', () => {
```
Comment (edgibbs) [inline src/test/Ethnicity/EthnicityForm_test.js]: small typo 'laitioOrigin' should be latinoOrigin

File: README.md [hunk 3]
```diff
@@ -19,24 +19,27 @@ The **react-wood-duck component library** is a set of re-usable components and s
 * [Storybook](#storybook)
```
Comment (edgibbs) [inline README.md]: There's a lot of changes to the README, but most of them look like spacing our use of an _, does it clean up some issues?

Comment (Sushmareddy-OSI) [reply README.md]: removed '_' and given '*' for line spacing

File: stories/components/Ethnicity/EthnicityForm.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
```
Comment (edgibbs) [inline stories/components/Ethnicity/EthnicityForm.js]: withInfo isn't used, but probably needs some lightweight docs

File: src/test/EthnicityForm_test.js [hunk 5]
```diff
@@ -0,0 +1,390 @@
+import React from 'react';
```
Comment (edgibbs) [inline src/test/EthnicityForm_test.js]: Tests look pretty comprehensive 😄

File: stories/components/Ethnicity/EthnicityForm.js (hunk 4 above)
Comment (Sushmareddy-OSI) [reply stories/components/Ethnicity/EthnicityForm.js]: added!

**Comment by edgibbs** [comment]
Code climate as just showing a few false positive duplications.

**Pull request merged**